hf-llm-inference/Dockerfile

```
@@ -0,0 +1,21 @@
+FROM ghcr.io/coreweave/ml-containers/torch:afecfe9-base-cuda12.0.1-torch2.0.0-vision0.15.1
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt update && apt upgrade -y && \
+    apt update && apt install -y git curl && \
+    apt clean
```

Didn't notice the `apt upgrade`, definitely don't want that. That will mess with all sorts of packages.
```suggestion
RUN apt-get -qq update && \
    apt-get -qq install --no-install-recommends -y git curl && \
    apt-get clean
```